frontend/src/lib/apiClient.js

```diff
@@ -15,6 +15,26 @@ let getRecipes = search => {
     .catch(e => { return [] });
 }; 
 
+let createUser = (data) => { //this works
+  return apiClient.post("api/v1/user", {
+    user: {
+      first_name: data.firstName,
+      last_name: data.lastName,
+      email: data.email,
+      password: data.password
+    }
+  })
+};
+
+let loginUser = (data) => {
+  let credentials = {email: data.email, password: data.password};
+  return apiClient.post("api/v1/auth/login", {
+    user: credentials
+  }).then(response => { 
+    debugger
+    console.log(response) })
```

Comment:
Still have some debugging stuff here.

Reply:
Yeah. I had started working on the sign up form. I can get rid of this for this pr

Reply:
I just got rid of `debugger` I'm going to branch off of this for sign in, so I kept the rest of the api call.